libs/types/src/permissions.rs

```diff
@@ -93,6 +94,7 @@ bitflags::bitflags! {
 		const LIQUIDITY_ADMIN = 0b00001000;
 		const MEMBER_LIST_ADMIN = 0b00010000;
 		const RISK_ADMIN = 0b00100000;
+		const POD_ADMIN = 0b01000000;
```

Comment:
I think this is a bit confusing as it's not an admin role, would just do:
```suggestion
		const POD_READ_ACCESS = 0b01000000;
```

Reply:
Agree, changed it.